docs/getting_started/bundles_starter_pack.md

```diff
@@ -0,0 +1,51 @@
+# Bundles starter pack
+
+eZ Platform application is a collection of bundles, similar to Symfony where ["everything is a bundle"](http://symfony.com/doc/current/book/bundles.html). Bundles listed below are useful additions to your clean eZ installation by using them you can improve your Platform usage.
+
+For more information about eZ bundle structure checkout:
+
+- [Bundle section](../guide/bundles.md) in our Guide.
+- List of [external bundles](https://ezplatform.com/Bundles) compatible with eZ Platform.
+
+## Everyday Platform usage
+
+|Bundle|Description|
+|------|-----------|
+|[ezmigrationbundle](https://github.com/kaliop-uk/ezmigrationbundle)|helps to handle content upgrades/migrations|
```

Comment:
can skip the ´s´ on upgrade and migration here I think, it is plural, but you only do once at a time

Or wdyt @DominikaK ?